source/egg/math/Quat.cc

```diff
@@ -80,6 +87,16 @@ f32 Quatf::dot(const Quatf &q) const {
     return w * q.w + v.dot(q.v);
 }
 
+void Quatf::setAxisRotation(f32 angle, const EGG::Vector3f &axis) {
+    const f32 half_angle = angle * 0.5f;
+    const f32 cos = Mathf::cos(half_angle);
+    const f32 sin = Mathf::sin(half_angle);
+
+    w = cos;
+    v = axis;
+    v *= sin;
```

Comment:
Probably want to rename `cos` and `sin` as these names are ambiguous with function names